internal/internal_task_pollers.go

```diff
@@ -397,8 +404,7 @@ func (wtp *workflowTaskPoller) processResetStickinessTask(rst *resetStickinessTa
 	return nil
 }
 
-func (wtp *workflowTaskPoller) RespondTaskCompletedWithMetrics(completedRequest interface{}, taskErr error, task *s.PollForDecisionTaskResponse, startTime time.Time) (response *s.RespondDecisionTaskCompletedResponse, err error) {
-
+func (wtp *workflowTaskPoller) RespondTaskCompleted(completedRequest interface{}, taskErr error, task *s.PollForDecisionTaskResponse, startTime time.Time) (response *s.RespondDecisionTaskCompletedResponse, err error) {
```

Comment:
not entirely sure how I feel about kinda-shadowing an internal helper method with a "public" method like this 🤔 
- it's rather clear they're related, and public is "preferred", which is nice
- there are a lot of case-insensitive searches all over, and people seem not all that likely to notice `r` vs `R`, so this seems like it might lead to a bit more false-matches / easy to miss during code review / etc

dunno.  do we have any other examples in this repo?  I can't think of any at the moment at least.  it might be a bit nicer than using `respondTaskCompletedInternal` or something if we like it in general.

Reply:
Renamed it back to RespondTaskCompletedWithMetrics.